sympy/core/tests/test_wester.py

```diff
@@ -75,7 +75,7 @@ def test_B2():
     d, e = FiniteSet(i), FiniteSet(j, k, l)
 
     assert (FiniteSet(i, j, j, k, k, k) & FiniteSet(l, k, j) &
-            FiniteSet(j, m, j)) == Union(a, Intersection(b, Union(c, Intersection(d, e))))
+            FiniteSet(j, m, j)) == Union(a, Intersection(b, Union(c, Intersection(d, FiniteSet(l)))))
```

Comment:
I verified using `.subs` for values from `0 to 5`(for all of `i, j, k, l, m`) and checking the values resulting and found this gave the same answer as the previous one for each of the `substitution`:
```
>>> b = []
>>> a = []
>>> c = []
>>> for x in range(0, 5):
           for y in range(0, 5):
               for z in range(0, 5):
                   for t in range(0, 5):
                       for f in range(0, 5):
                           c.append(t1_2.subs({i: x, j: y, k: z, l: t, m: f}) == t2.subs({i: x, j: y, k: z, l: t, m: f}))
>>> False in c
False
```
where `t2` is the answer generated by current master and `t1_2` is the set generated by my commit.